spanner_orm/field.py

```diff
@@ -194,19 +206,26 @@ def validate_type(value) -> None:
 
   @staticmethod
   def matches(type: str) -> bool:
-    return type.startswith('STRING(') and type.endswith(')')
+    val = re.findall('STRING\((.*)\)', type)
```

Comment:
Shouldn't this be re.fullmatch(), because the string has to be the type exactly with nothing else?